bm-instance-agent/bm_instance_agent/manager.py

```
@@ -173,3 +193,368 @@ def console(self):
+        gpu_type = ""
+        gpu_devs = []
+        r, o, e = bm_utils.shell_cmd("lspci -Dmmnnv", False)
+        if r != 0:
+            return
+        # parse lspci output
+        for part in o.split('\n\n'):
+            vendor_name = ""
+            device_name = ""
+            sub_vendor_name = ""
+            for line in part.split('\n'):
+                if len(line.split(':')) < 2: continue
```

Ruff (0.12.2)

Multiple statements on one line (colon)

(E701)